src/rtpghi~.c

```diff
@@ -12,80 +12,65 @@
 static t_class *rtpghi_tilde_class;
 
 typedef struct _rtpghi_tilde {
-    t_object  x_obj;
-    t_float W_pd;
-    t_float ol_pd;
-    t_float M_pd;
-    t_float tol_pd;
-    t_float do_causal_pd;
-    t_symbol window_type;
-    phaseret_rtpghi_state_s* sta_pd;
-    ltfat_complex_s *c;
+	t_object  x_obj;
+	t_float ol_pd;
+	t_float M_pd;
+	t_float tol_pd;
+	t_float do_causal_pd;
+	t_symbol window_type;
```

Comment:
that should be a point to a symbol (`t_symbol*window_type;`)

also i'm not exactly sure about the naming scheme (why is there no `_pd` suffix?)